### PR TITLE
chore: add comments for bot bicep files

### PR DESCRIPTION
**Pull request by blackchoey**
Add comments for bot bicep files so users can understand the provision behavior easier

### CODE REVIEW

File: packages/fx-core/templates/plugins/resource/bot/bicep/botProvision.template.bicep [hunk 1]
```diff
@@ -3,14 +3,15 @@ param provisionParameters object
...
+var botServiceName = contains(provisionParameters, 'botServiceName') ? provisionParameters['botServiceName'] : '${resourceBaseName}' // Try to read name for Azure Bot Service from parameters
+var botServiceSku = contains(provisionParameters, 'botServiceSku') ? provisionParameters['botServiceSku'] : 'F0' // Try to read SKU for Azure Bot Service from parameters
+var botDisplayName = contains(provisionParameters, 'botDisplayName') ? provisionParameters['botDisplayName'] : '${resourceBaseName}' // Try to read display name for Azure Bot Service from parameters
+var serverfarmsName = contains(provisionParameters, 'botServerfarmsName') ? provisionParameters['botServerfarmsName'] : '${resourceBaseName}bot' // Try to read name for App Service Plan from parameters
+var webAppSKU = contains(provisionParameters, 'botWebAppSKU') ? provisionParameters['botWebAppSKU'] : 'F1' // Try to read SKU for Azure Web App from parameters
+var webAppName = contains(provisionParameters, 'botSitesName') ? provisionParameters['botSitesName'] : '${resourceBaseName}bot' // Try to read name for Azure Web App from parameters
 
+// Registers your web service as a bot with the Bot Framework
```
Comment (YitongFeng-git) [inline packages/fx-core/templates/plugins/resource/bot/bicep/botProvision.template.bicep:14]: Register

File: packages/fx-core/templates/plugins/resource/bot/bicep/botProvision.template.bicep [hunk 2]
```diff
@@ -25,6 +26,7 @@ resource botService 'Microsoft.BotService/botServices@2021-03-01' = {
   }
 }
 
+// Connects the bot service to Microsoft Teams
```
Comment (YitongFeng-git) [inline packages/fx-core/templates/plugins/resource/bot/bicep/botProvision.template.bicep:29]: Connect

File: packages/fx-core/templates/plugins/resource/bot/bicep/botProvision.template.bicep [hunk 3]
```diff
@@ -43,30 +46,31 @@ resource serverfarm 'Microsoft.Web/serverfarms@2021-02-01' = {
   }
 }
 
+// Web app that hosts your bot
```
Comment (YitongFeng-git) [inline packages/fx-core/templates/plugins/resource/bot/bicep/botProvision.template.bicep:49]: It's better to user proper noun, "Web App"

**Review by YitongFeng-git — DISMISSED** [review]

File: packages/fx-core/templates/plugins/resource/bot/bicep/botProvision.template.bicep (hunk 1 above)
Comment (blackchoey) [reply packages/fx-core/templates/plugins/resource/bot/bicep/botProvision.template.bicep]: Thanks, resolved all the comments

**Review by YitongFeng-git — APPROVED** [review]

**Pull request merged**